src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: "Roboto" , sans-serif;
+  font-style: normal;
+}
+
+.header-content {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  align-items: center;
+  padding: 0 50px;
+  max-width: 100%;
+}
+
+.logo {
+  padding-top: 5px;
+}
+
+.menu-moyo-list {
+  display: flex;
+}
+
+.nav-moyo-item {
+  padding-left: 20px;
+  list-style: none;
+  display: flex;
+}
+
+.nav-moyo-item:first-child {
+  padding-left: 0;
+}
+
+.nav-moyo-link {
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: #000;
+  padding: 23px 0;
+  transition: all 0.5s;
```

Comment:
Don't apply the transition to all properties. Specify the exact properties
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x